messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -210,16 +287,53 @@ private void releaseToken(Segment segment) {
         }
     }
 
+    /**
+     * Indicates whether the {@code eventMessage} identified with given {@code token} should be processed as part of the
```

Comment:
The javadoc might be a little clearer if it stated:
`Indicates whether the {@code eventMessage} indentifier with the given {@code token} **it's current position** should be...`